.github/workflows/workflow-common.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         continue-on-error: true
         run: ./common_dev/scripts/cyclomatic.py
       - name: Run pylint
-        run: ./common_dev/scripts/pylintchecker.py -c scargo/ -s 9.9 --exclude=tests/
+        run: ./common_dev/scripts/pylintchecker.py -c scargo/ -s 9.95 --exclude=tests/
```

Comment:
I will rather suggest not increasing the minimum score each time when we add fixing old code. Reasoning: I would like to avoid blockers which may be related to the linter only (e.g. using _() for multilanguage support) against progress in work.
I suggest keeping score as high as possible, but keep "the bar" at a certain constant minimum

Reply:
I raised the score by 0.07, but the bar only by 0.05, in my opinion this is reasonable. If we don't block on linters, we might as well not be linting at all.